webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  entry: {
+    'js/search/timetable': './js/timetable.js.jsx'
```

Comment:
I might be missing something, but isn't this file actually in `./public/js/...`?